src/components/00-materials/story.js

```diff
@@ -194,8 +264,15 @@ export const IconsAndImages = () => {
         margin: 20px 0;
       }
 
-      .item-name {
+      .materials__asset-name {
         margin-left: 5px;
+        margin-right: 10px;
+        width: 210px;
```

Comment:
We have enough space on the right. Maybe just not set a width to this row (at line 270)

Reply:
Yeah, this is a good idea I think